### ISSUE TITLE
Using "u" for the micro symbol

### ISSUE THREAD

**Issue by jrhim** [sebkopf/chemtools#17]
Could we have the alphabet "u" recognized as the micro symbol (e.g., uL = microliter)? Or is there a different way to express 'micro'? Thank you!

**Comment by sebkopf** [comment]
good idea, i'll take a look. in the meantime if you're on Mac, you can use `Option`+`m` to type µ (I don't know if there is a windows shortcut)

**Comment by jrhim** [comment]
Thank you! The 'option + m' is a very helpful tip in general (I use Mac). 👍

**Comment by sebkopf** [comment]
#10 should take care of allowing `m`, `µ`, and `micro` all as valid prefixes
